src/shared/locales/vi/about.js

```diff
@@ -0,0 +1,7 @@
+export default {
+    'engine-version': 'Phiên bản Ứng dụng',
+    'license': 'Giấy phép',
+    'about': 'Về Motrix',
+    'release': 'Phát hành',
+    'support': 'Hỗ trợ'
+  }
```

Comment:
Please keep the indentation consistent with the files in other locales

- Use 2 spaces
- The } at the end adheres to the beginning of the line
- Add a blank line at the end of the file

Several other files have also changed the format, thanks

Reply:
@agalwood Thank you, I have updated.